spring-boot-starter-camunda-sdk/src/test/java/io/camunda/zeebe/spring/client/config/CredentialsProviderSaasTest.java

```diff
@@ -51,20 +52,30 @@
       "camunda.client.cluster-id=12345",
       "camunda.client.region=bru-2",
       "camunda.client.auth.client-id=my-client-id",
-      "camunda.client.auth.client-secret=my-client-secret",
-      "camunda.client.auth.issuer=http://localhost:14682/auth-server"
+      "camunda.client.auth.client-secret=my-client-secret"
     })
 @EnableConfigurationProperties({
   ZeebeClientConfigurationProperties.class,
   CamundaClientProperties.class
 })
-@WireMockTest(httpPort = 14682)
 public class CredentialsProviderSaasTest {
 
+  @RegisterExtension
+  static WireMockExtension wm =
+      WireMockExtension.newInstance()
+          // dynamic port is the default
```

Comment:
🔧 Let's be specific and directly use the dynamic, with that we make sure that it works even if they change the defaults and we don't need to document it.